src/document_array_construct.py

```diff
@@ -0,0 +1,57 @@
+import os
+import uuid
+
+from faker import Faker
+
+from jina import Document, DocumentArray, __version__
+from jina.types.arrays.memmap import DocumentArrayMemmap
+from jina.logging.profile import TimeContext
```

Comment:
this is annoying to use, too verbose. I created a different TimeContext instance without the noise

Reply:
okay